### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by jaimevillegas**
I have made the following changes to the code:

- Configured Webpack with all dependencies and templates.
- Created `index.html` and added content
- Created `index.js` with a class `Task` that will do all the operations of the tasks
- Created `style.css` and added styling to the website
- Created `README.md` and `LICENSE.md`

Thank you for taking the time to review my code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    this.completed = completed;
+    this.id = id;
+  }
+
+  static tasks = [];
+
+  updateList() {
+    this.foo = 'eslint use this';
+    tasksList.innerHTML = '';
+    Task.tasks.forEach((task) => {
```
Comment (brytebee) [inline src/index.js]: - [x] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.id - b.id)`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @jaimevillegas👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jaimevillegas) [reply src/index.js]: @brytebee Thank you for your comment!
I have worked on that!

**Review by sumairq — APPROVED** [review]
### Hi @jaimevillegas , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**